packages/commonwealth/server/farcaster/frames/contest/checkEligibility.tsx

```diff
@@ -1,11 +1,58 @@
 import { Button } from 'frames.js/express';
 import React from 'react';
 import { frames } from '../../config';
-import { CardWithText } from '../../utils';
+import { circleCheckIcon, circleXIcon, fakeApiCall } from '../../utils';
 
 export const checkEligibility = frames(async () => {
+  let eligible: boolean;
+
+  try {
+    await fakeApiCall({ error: 'error' });
```

Comment:
will the data be server-side rendered or fetched by the client?

Reply:
1. fakeApiCall is just for testing purpose and it will be deleted soon
2. the whole idea is that our endpoint returns object with markup, title and additional necessary fields. Then all of it is transformed to the static image, which eventually will be displayed as a frame in farcaster client. So I would say its server-side rendered but not in the sense we all know what that means